### PR TITLE
Add linux distrubtion PHP version defaults

### PR DESCRIPTION
**Pull request by bcrowe**

### CODE REVIEW

File: _data/linux_distros.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  default: 5.3.2
+
+- name: "CentOS 7.0"
+  default: 5.4.16
+
+- name: "CentOS 6.6"
+  default: 5.3.3
+
+- name: "CentOS 5.11"
+  default: 5.1.6
```
Comment (GrahamCampbell) [inline _data/linux_distros.yml]: Who still uses this!

Comment (bcrowe) [inline _data/linux_distros.yml]: Definitely a tad bit old... but RHEL/CentOS have 10 year support cycles, so 5 will be supported for another couple years.

Comment (philsturgeon) [inline _data/linux_distros.yml]: It is sadly very relevant.

Comment (GrahamCampbell) [inline _data/linux_distros.yml]: Sad, yeh.

**Pull request merged**